cimbar/util/bit_file.py

```diff
@@ -6,28 +6,31 @@
 
 
 class bit_file:
-    def __init__(self, f, bits_per_op, mode='read', read_size=MAX_ENCODING):
+    def __init__(self, f, bits_per_op, mode='read', keep_open=False, read_size=MAX_ENCODING, read_count=1):
```

Comment:
The `read_size` + `read_count` thing is pretty weird, and I'm not really happy with it.

But the premise here is to give `bit_file` a clear eof condition for the fountain encoding case. We could generate fountain chunks essentially indefinitely (not quite, but long enough to many any sentient being impatient)...